### ISSUE TITLE
[regression in 3.12.13] mkdir: /usr/local/share/coin/doc: Permission denied

### ISSUE THREAD

**Issue by yurivict** [coin-or/Ipopt#339]
It seems that version ```3.12.13``` now builds the documentation and then fails to install it into the correct location:
```
Running dot...
lookup cache used 13963/65536 hits=134101 misses=15308
finished...
/usr/ports/math/ipopt/work/Ipopt-3.12.13/install-sh -d /usr/local/share/coin/doc/Ipopt/doxydoc
mkdir: /usr/local/share/coin/doc: Permission denied
*** Error code 1
```

It looks like ```$(DESTDIR)``` is missing in the install instruction in the makefile.

**Comment by svigerske** [comment]
I believe to have fixes this. At least I found some places where `$(DESTDIR)` seems to have been missing. That should make it into the next release.

**Comment by yurivict** [comment]
This still isn't fixed. This patch is needed (on the Makefile.in level):
```
--- Makefile.in.orig    2019-04-29 04:09:03 UTC
+++ Makefile.in
@@ -711,8 +711,8 @@ clean-doxydoc :
        cd doxydoc && rm -rf html *.log *.tag
 
 install-doxydoc : doxydoc 
-       $(mkdir_p) $(DocInstallDir)/doxydoc
-       cp -R doxydoc/html doxydoc/ipopt_doxy.tag $(DocInstallDir)/doxydoc
+       $(mkdir_p) $(DESTDIR)$(DocInstallDir)/doxydoc
+       cp -R doxydoc/html doxydoc/ipopt_doxy.tag $(DESTDIR)$(DocInstallDir)/doxydoc
 
 uninstall-doxydoc:
        if test -e $(DocInstallDir)/doxydoc ; then rm -rf $(DocInstallDir)/doxydoc ; fi
```

**Comment by svigerske** [comment]
Yes, I would make this available with 3.12.14.
Can you work around this for now?

**Comment by yurivict** [comment]
> Can you work around this for now?


Yes!
